### PR TITLE
chore: replace todo with error

### PR DESCRIPTION
**Pull request by sundy-li**
1. add DataType::LargeList path in match
2. replace todo with error (py runtime will hang if panic happens by todo)

### CODE REVIEW

File: arrow-udf-js-deno/src/deno_arrow.rs [hunk 1]
```diff
@@ -744,7 +744,9 @@ impl Converter {
                 }
                 Ok(Arc::new(builder.finish()))
             }
-            DataType::FixedSizeBinary(_) => todo!(),
+            other @ DataType::FixedSizeBinary(_) => {
```
Comment (Xuanwo) [inline arrow-udf-js-deno/src/deno_arrow.rs:747]: Maybe we can remove this line directly?

File: arrow-udf-js-deno/src/deno_arrow.rs [hunk 2]
```diff
@@ -1207,7 +1201,9 @@ impl Converter {
                     }
                 }
             }
-            DataType::FixedSizeBinary(_) => todo!(),
+            other @ DataType::FixedSizeBinary(_) => {
```
Comment (Xuanwo) [inline arrow-udf-js-deno/src/deno_arrow.rs:1204]: The same.

**Review by Xuanwo — COMMENTED** [review]
Please format the code.

**Comment by Xuanwo** [comment]
Oh, sorry. The check is not about `fmt`:

```shell
error: transmute used without annotations
   --> arrow-udf/src/ffi.rs:184:40
    |
184 |     let input_ref = unsafe { std::mem::transmute(input.as_ref()) };
    |                                        ^^^^^^^^^ help: consider adding missing annotations: `transmute::<&arrow_array::RecordBatch, &arrow_array::RecordBatch>`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#missing_transmute_annotations
    = note: `-D clippy::missing-transmute-annotations` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::missing_transmute_annotations)]`
```

**Review by wangrunji0408 — APPROVED** [review]
LGTM! Thank you.

May I release new versions after merging this PR?

**Comment by Xuanwo** [comment]
> May I release new versions after merging this PR?

That would be greatly appreciated.

**Pull request merged**